src/app/pokemons/[name]/page.tsx

```diff
@@ -0,0 +1,65 @@
+import Card from '@/components/Card';
+import { fetchPokemonDetails } from '../../../../lib/pokemons';
+import { IPokemonDetailPage } from './types';
+
+const styles = {
+  card: 'p-4 m-4 shadow-sky-500 border border-sky-500',
+  img: 'w-1/10 p-4 isplay: inline',
+  container: 'py-0.5 px-4',
+  grid: 'grid gap-4 grid-cols-2 grid-rows-3 text-left',
+};
+
+export default async function Page(props: IPokemonDetailPage) {
+  const { params } = props;
+  const { name } = params;
+
+  const data = await fetchPokemonDetails(name);
+
+  console.log('data', data);
```

Comment:
Make sure to remove console logs before pushing your code. You can configure eslint to handle that for you. Also, you can run the eslint in the pre-commit hook so that these checks are executed before commiting your changes. This ensures that your codebase is clean